server/examples/config-ldes-all.json

```diff
@@ -75,7 +75,7 @@
                   "prefix": "default",
                   "view": {
                     "@type": "MongoSDSView",
-                    "descriptionId": "http://localhost:3000/ldes/#timestampFragmentation",
+                    "descriptionId": "http://127.0.0.1:3000/ldes/#timestampFragmentation",
```

Comment:
This is an identifier, so I don't think it makes sense to change it to 127.0.0.1.